README.md

```diff
@@ -19,7 +19,7 @@ Current features:
 
 Supported versions:
 
-*  Kubernetes: 1.11+
+*  Kubernetes 1.12+ or OpenShift 3.11+
 *  Elasticsearch: 6.8+, 7.1+
```

Comment:
I wonder if `Elastic stack` would be more appropriate than `Elasticsearch` here?